frontend/src/app/manage-subscription/page.tsx

```
@@ -479,6 +761,202 @@ export default function ManageSubscriptionPage() {
+            </div>
+          </motion.div>
+        </div>
+      )}
+
+      {/* Downgrade Confirmation Modal */}
+      {showDowngradeConfirmation && (
+        <div 
+          className="fixed inset-0 z-50 flex items-center justify-center p-4 bg-black/50 backdrop-blur-sm"
+          onClick={() => !processingAction && setShowDowngradeConfirmation(false)}
+        >
+          <motion.div
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)